core/src/test/resources/test_variant_summaries.tsv

```diff
@@ -30,7 +30,7 @@ BRCA1	T100*	Likely Oncogenic	The BRCA1 T100* is a truncating mutation in a tumor
 ### Truncating mutations in TSG that are not curated
 PPP2R2A	T100*	Unknown	The PPP2R2A T100* mutation has not specifically been reviewed by the OncoKB team, and therefore its biological significance is unknown.
 ### Truncating mutation in both
-GATA3	T100*	Likely Oncogenic	The GATA3 T100* is a truncating mutation; truncating mutations in this gene are considered likely oncogenic.
```

Comment:
Let's use gene `PTPN1`. The Truncating Mutations was removed from GATA3 in this release. The purpose of the test is to use oncogenicity from Truncating Mutations.